### PR TITLE
Rewrite namespace details view

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- 

**Related issue(s)**

### CODE REVIEW

**Comment by parostatkiem** [comment]
- Didn't the labels editing component have to be inline, not a modal? 
image
- Please move the question mark a bit further from the "Labels" word
image

File: core-ui/src/components/NamespaceDetails/DeployResourceModal/DeployResourceModal.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        method: 'POST',
+        body: JSON.stringify(content),
+        headers: {
+          Authorization: `Bearer ${idToken}`,
+          'Content-Type': 'application/json',
+        },
+      });
+    } catch (e) {
+      console.warn(e);
+      LuigiClient.uxManager().showAlert({
```
Comment (sjanota) [inline core-ui/src/components/NamespaceDetails/DeployResourceModal/DeployResourceModal.js:47]: NotificationContext?

**Pull request merged**